libraries/AP_HAL_PX4/Util.cpp

```diff
@@ -236,15 +242,16 @@ extern "C" {
 */
 void *PX4Util::dma_allocate(size_t size)
 {
-#ifndef CONFIG_ARCH_BOARD_PX4FMU_V1
+#if !defined(CONFIG_ARCH_BOARD_PX4FMU_V1)
```

Comment:
this seems to end up with fat_dma_alloc() but uses free(). That doesn't sound safe